src/dev-mock-service/README.md

```diff
@@ -0,0 +1,47 @@
+## Dev Mock Service
+
+Dev Mock Service Worker is an API mocking library that allows you to specify
```

Comment:
This isn't actually a server worker is it?

Reply:
Whoops, bad copy paste 😅